js/views/chatVw.js

```diff
@@ -113,9 +130,13 @@ module.exports = baseVw.extend({
         filteredMd = this.filteredChatConvos.findWhere({ guid: md.get('guid') });
         filteredMd && filteredMd.set(md.attributes);
       }
-
-      md.hasChanged('unread') && this.setAggregateUnreadCount();          
-    });      
+      
+      if (md.hasChanged('unread') && md.get('unread') > 0) {
```

Comment:
We need to resort otherwise you have no way of moving unread messages to top other than doing it manually in DOM, which is ugly.

I'll check `> 0` again. To tell you the truth I don't even remember why I added it, this part was done more than a week ago.